### PR TITLE
correct level one errors for line 7/8/25/35 for installation overview

### PR DESCRIPTION
**Pull request by Amrita42**
- Applies to main and 4.11.
- Rectifies the level one issues for the installation.overview file (merged PR https://github.com/openshift/openshift-docs/pull/46421)
- [Preview](http://file.del.redhat.com/asakthiv/Commonterms-install-asciierrrs/installing/index.html) 
- @jeana-redhat ptal, needs a merge asap.

### CODE REVIEW

**Review by tmalove — COMMENTED** [review]
LGTM

File: modules/installation-overview.adoc [hunk 1]
```diff
@@ -6,6 +6,8 @@
 :_content-type: CONCEPT
 [id="installation-overview_{context}"]
```
Comment (jeana-redhat) [inline modules/installation-overview.adoc:8]: Remove line 8 so there is no space between the id and the heading (https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata)

**Review by jeana-redhat — COMMENTED** [review]
One minor structural thing. Can you please link a preview for verification? (Screenshot of a local build is fine if you don't want to mess with an upload :+1:)

**Pull request merged**